solidity/random-beacon/README.adoc

```diff
@@ -55,39 +55,38 @@ Instead of a v1 ticket-based approach for group selection, we use a sortition
+the result to the chain calling
+`RandomBeacon.submitDkgResult(DKG.Result calldata dkgResult)` function.
+Once the result is submitted, a challenge period starts.
 
 During the challenge period, anyone can notify that the submitted DKG result is
-malicious calling `RandomBeacon.challengeDkgResult(DKG.Result calldata dkgResult)`
+malicious by calling `RandomBeacon.challengeDkgResult(DKG.Result calldata dkgResult)`
 function. A malicious DKG result may contain corrupted data, group members not
 selected by the pool, or incorrect supporting signatures. If such malicious
-result is submitted and successfully challenged, the result submitter gets slashed and the
-malicious result is immediately discarded.
-
-Once the challenge period passes, and no challenges are reported,
-the DKG result submitter should unlock the sortition pool and mark the DKG result as
-accepted calling
-`RandomBeacon.approveDkgResult(DKG.Result calldata dkgResult)` to receive a
-reward. In case the submitter does not call the approve function within a
-specific number of blocks, anyone can do that and receive the
-submitter's reward as described in <<fees-and-rewards,Fees and Rewards>> section.
-
-There is a timeout before which a DKG result should be submitted. The timeout
-equals the group size multiplied by the number of blocks for a member to become
-eligible to submit a DKG result. The timer starts at the moment when the first
-member becomes eligible.
-
+result is submitted and successfully challenged, the result submitter gets
+slashed and the malicious result is immediately discarded.
+
+Once the challenge period passes, and no valid challenge is reported, the DKG result
+submitter should unlock the sortition pool and mark the DKG result as approved
+calling `RandomBeacon.approveDkgResult(DKG.Result calldata dkgResult)`
+The submitter receives  eth reimbursement for both `submitDkgResult` and
+`approveDkgResult` transaction as described in
+<<transaction-incentives,Transaction Incentives>> section. In case the original
+submitter does not call the `approveDkgResult` function within a specific number
+of blocks, anyone can do that and receive the submitter's reimbursement.
+
+There is a timeout before which a DKG result should be submitted.
 In case the DKG result was not submitted before a timeout, anyone can notify DKG
-timed out calling `RandomBeacon.notifyDkgTimeout()` and receive a reward, as
-described in <<fees-and-rewards,Fees and Rewards>> section. DKG timeout includes
-the situation when no new relay entry was produced and sortition could not be
-performed.
+timed out by calling `RandomBeacon.notifyDkgTimeout()` and unlock the sortition
+pool. DKG timeout includes the situation when no new relay entry was produced
+and sortition could not be performed.
+
+Off-chain clients are expected to follow the submission order when submitting
+DKG result to avoid front-running and minimize the cost but no ordering is
+enforced on-chain.
```

Comment:
Could we link 'submission order' to the Operator-Only section in Transaction Incentives?

Reply:
1fadd0bedacc1e8332f6ef3c24e1f6db4acc3322